src/frontend/staff-portal/src/app/components/contact-info/contact-info.component.ts

```diff
@@ -260,9 +260,9 @@ export class ContactInfoComponent implements OnInit {
 
       // set violation date and time
       let tempViolationDate = new Date(this.lastUpdatedDispute.issuedDate);
+      tempViolationDate.setHours(tempViolationDate.getHours() + 7);
```

Comment:
Can you please help to test removing the "new Date" and this line like below
let tempViolationDate = this.lastUpdatedDispute.issuedDate;